src/kademlia/find_data.cpp

```diff
@@ -135,8 +135,6 @@ char const* find_data::name() const { return "find_data"; }
 
 void find_data::done()
 {
-	if (m_invoke_count != 0) return;
```

Comment:
this makes it look like done() may be called when a single lookup is done, and it's up to find_data::done() to make sure it only passes on the call to traversal_algorithm::done() once. is m_invoke_count always 0 here?